br/pkg/pdutil/pd.go

```diff
@@ -95,6 +95,8 @@ var (
 		"shuffle-leader-scheduler":     {},
 		"shuffle-region-scheduler":     {},
 		"shuffle-hot-region-scheduler": {},
+
+		"evict-slow-store-scheduler": {},
```

Comment:
LGTM, but what if PD add new scheduler in future?

Reply:
PD hasn't the list of all the scheduler.